app/assets/javascripts/helpers/api_routes.js

```diff
@@ -14,13 +14,14 @@ class ApiRoutes {
 
   get artists() {
     return {
-      index: `/api/artists`,
-      show: id => `/api/artists/${id}`,
-      update: id => `/api/artists/${id}`,
-      delete: id => `/api/artists/${id}`,
-      works: id => `/api/artists/works/${id}`,
-      requests: id => `/api/artists/requests/${id}`
-    };
+      show        : (id) => `/api/artists/${id}`,
```

Comment:
I think ur missing the
index: `api/artists`
route

Reply:
you don't need the AllArtists route below since you're retrieving the route from `APIRoutes.artists.index`